delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/Files.scala

```diff
@@ -229,6 +229,33 @@ final class Files(
     } yield res
   }.span("updateFileMetadata")
 
+  def registerFile(
```

Comment:
Not sure about the register keyword, I don't know if there could be something clearer ?

Reply:
I thought "link" seemed clearer and more accurate than its current usage, but has too much historical baggage 😅

Reply:
What about something like bind ?